src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -335,7 +331,6 @@ void GetZoneAirSetPoints(EnergyPlusData &state)
     auto &ComfortTStatObjects = state.dataZoneCtrls->ComfortTStatObjects;
     auto &ComfortControlledZone = state.dataZoneCtrls->ComfortControlledZone;
     int NumOfZones = state.dataGlobal->NumOfZones;
-    auto &StageControlledZone = state.dataZoneCtrls->StageControlledZone;
```

Comment:
Why eliminate this bad array shortcut and not the other ones around it?

Reply:
Because this change eliminated a CppCheck style suggestion and reducing these takes time and risks diffs when a mistake is made. These are the types of changes that will occur after reducing the number of suggestions to a manageable level.